### ISSUE TITLE
Unlabelled buttons in the Amazon Q VSCode extension

### ISSUE THREAD

**Issue by DavidJ1001** [aws/aws-toolkit-vscode#7529]
## Problem
When in the Amazon Q Chat Panel and navigating with the JAWS Screen Reader the buttons are announced as unlabelled and I am unable to easily identify what feature the button is meant for.
## System details (run `AWS: About` and/or `Amazon Q: About`)

-   OS: Linux x64 6.6.87.2-microsoft-standard-WSL2
-   Visual Studio Code extension host:  1.101.0
-   Amazon Q version: 1.77.0
